Application/EdFi.Ods.Api/Security/Authorization/ResourceClaimUriProvider.cs

```diff
@@ -54,15 +80,22 @@ public string[] GetResourceClaimUris(Type resourceType)
 
                     var schemaNameMap = schemaNameMapProvider.GetSchemaMapByProperCaseName(schemaProperCaseName);
 
-                    var uris = new[]
-                    {
-                        // Schema-based URI format
-                        CreateSchemaBasedResourceClaimUri(schemaNameMap.UriSegment, resourceName),
+                    // Include legacy representation for Ed-Fi Standard resources only
```

Comment:
As written, this code was providing support for extension resources to use the URI format that does not include the schema segment. This runs counter to the guidance (and intent) regarding extension resources. Going forward, we will not be supporting matching extension resources to claim URIs without the schema appropriately included.